lib/DiscoveryStreamFeed.jsm

```diff
@@ -851,9 +851,10 @@ this.DiscoveryStreamFeed = class DiscoveryStreamFeed {
    */
   async refreshAll(options = {}) {
     const { updateOpenTabs, isStartup } = options;
+    const dispatchQueue = [];
```

Comment:
Add a comment why we have a queue (probably something along the lines of simulating a transaction)

Reply:
:+1: